core/finality-grandpa/src/communication/mod.rs

```diff
@@ -145,8 +146,11 @@ impl<B, S, H> Network<B> for Arc<NetworkService<B, S, H>> where
 	S: network::specialization::NetworkSpecialization<B>,
 	H: network::ExHashT,
 {
-	type In = NetworkStream;
+	type In = NetworkStream<
+		Box<dyn Stream<Item = network_gossip::TopicNotification, Error = ()> + Send + 'static>,
+	>;
 
+	#[allow(deprecated)]
```

Comment:
What is deprecated in here?

Reply:
boxed

Reply:
Then just call `Box::new()`.

Reply:
Tried, but it didn't work here on rust nightly :(

Reply:
Why?

Reply:
Np, fixed